Integrations/integration-rasterize.yml

```diff
@@ -56,6 +56,45 @@ script:
             demisto.results(error_message)
         sys.exit(0)
 
+    if demisto.command() == 'rasterize-image':
+        res = demisto.getFilePath(demisto.args()['EntryID'])
+        # demisto.results({'res':res})
+        # res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
+        with open(res['path'], 'r') as f:
+            data = f.read()
+        b64 = base64.b64encode(data)
+        html = '<img src="data:image/png;base64, ' + b64 + '">'
+
+        return_code = 0
+        error_message = ''
+
+        friendlyName = 'image.png'
+        f = open('htmlImage.html','w')
+        f.write('<html style="background:white";>' + html + '</html>')
```

Comment:
`;` should be inside the style and not after `"`. Also, I would add `<body>` tags just to be nice.